openedx_events/apps.py

```diff
@@ -20,6 +22,10 @@ def general_signal_handler(sender, signal, **kwargs):  # pylint: disable=unused-
     #        "topic_a": { "event_key_field": "my.key.field", "enabled": True },
     #        "topic_b": { "event_key_field": "my.key.field", "enabled": False }
     # }"
+    if SIGNAL_PROCESSED_FROM_EVENT_BUS in kwargs and kwargs[SIGNAL_PROCESSED_FROM_EVENT_BUS] is True:
+        logger.info(f"Signal already processed on the Event Bus {signal.event_type}, skipping...")
```

Comment:
Not sure if this should be logged or if should be behind the `AUDIT_LOGGING` flag

Reply:
Yes, it is useful for debugging but this change is also related to the event bus implementation

Reply:
Would `logger.debug` be more appropriate? In certain cases, couldn't this be common and expected and just cause a lot of unnecessary logging?

Reply:
Using logger.debug would be better than introducing a new flag in the repository. So operators can still access these types of logs but have to configure the Python logger to do so.